templates/osd-spec.yml.j2

```diff
@@ -0,0 +1,10 @@
+data_devices:
```

Comment:
Doesn't cephadm detect all storage disks except the root disk automatically and use them for deployment? If yes, why do we need to pass this list from the templates?